test/tests/functional/pbs_alps_inventory_check_hook.py

```diff
@@ -118,38 +131,26 @@ def test_cray_login_nodes(self):
         """
         Test log when no nodes with vntype 'cray_login' are present.
         """
-        if self.platform == "craysim":
-            now = time.time()
-            mc = self.mom.parse_config()
-            save = mc["$alps_client"]
-            del mc["$alps_client"]
-            self.mom.apply_config(mc)
-            self.mom = self.moms.values()[0]
-            self.host = self.mom.shortname
-            rc = self.server.manager(MGR_CMD_DELETE, NODE, None, "")
-            self.assertEqual(rc, 0)
-            rc = self.server.manager(MGR_CMD_CREATE, NODE, id=self.host)
-            self.assertEqual(rc, 0)
+        now = time.time()

```

Comment:
You might want to int() this.  time.time() returns a float.  I've had issues where something happened very very fast and since pbs logs have second granularity, my message was not matched.  Same with the rest of the test.